x-pack/test/security_solution_endpoint/apps/integrations/artifact_entries_list.ts

```diff
@@ -51,10 +51,13 @@ export default ({ getPageObjects, getService }: FtrProviderContext) => {
       .delete(`${EXCEPTION_LIST_URL}?list_id=${listId}&namespace_type=agnostic`)
       .set('kbn-xsrf', 'true');
   };
-
+  // It's flaky only in Serverless
   // Failing: See https://github.com/elastic/kibana/issues/187314
   // Failing: See https://github.com/elastic/kibana/issues/187383
-  describe.skip('@ess @serverless For each artifact list under management', function () {
```

Comment:
I believe linked failures are from ESS not Serverless? 🤔

Reply:
Ok, I'll remove the `It's flaky only in Serverless` comment. This test is currently unskipped , and only running on ESS. Should I add `@serverless` to this test?